src/components/NotificationsModal/CollectionLink.tsx

```diff
@@ -0,0 +1,49 @@
+import { Route, route } from 'nextjs-routes';
```

Comment:
this component is the clickable link here 
<img width="336" alt="image" src="https://user-images.githubusercontent.com/6754223/200647521-5232e342-4028-4068-9cb9-d12e28c7143c.png">